tests/unit/test_client.py

```diff
@@ -24,7 +24,6 @@
 import unittest
 import warnings
 
-import freezegun
```

Comment:
Do you need to also remove freezegun installation from the noxfile?

Reply:
It is still used in `TestQueryJob.test_result_w_timeout()`, but an attentive remark nevertheless!